app/frontend/src/components/stories/Avatar.stories.tsx

```diff
@@ -0,0 +1,40 @@
+// also exported from '@storybook/react' if you can deal with breaking changes in 6.1
```

Comment:
I think it would be better to either:
- put this in the `components` folder as well, or
- move the `Avatar` component into its own folder named that, then put this stories file in that folder

so that the stories file is colocated with the component as recommended by Storybook themselves.

Reply:
Ok.